schemas/2019-09-07/Microsoft.Kusto.json

```diff
@@ -159,14 +161,26 @@
       ],
       "description": "Microsoft.Kusto/clusters/databases"
     },
-    "clusters_databases_dataconnections": {
+    "clusters_databases_dataConnections": {
       "type": "object",
+      "oneOf": [
```

Comment:
Is it ok that is not part of the properties ?

Reply:
Yes this is fine, since the properties declared inside the "oneOf" don't overlap with any of the properties defined in the outer scope.

The reason that it's been declared this way I think is because you're using a discriminator pattern on the "kind" value to allow polymorphism. See the following for example - for example - if the kind is set to "IotHub", then the appropriate iothub properties will be shown:
https://github.com/Azure/azure-resource-manager-schemas/blob/7bf8394c256f0dfbbc96764349bc6ad423f90c4d/schemas/2019-09-07/Microsoft.Kusto.json#L781-L786